test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,41 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env helps the user to set environmental variables which are needed force
+the usage of Minishift with docker command on VM host.
+INFO: This feature runs against persistent shell instance. To use non-default shell, please
+select one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_BASH parameter of make integration.
+
+  Scenario: Docker client is not set up
+  INFO: This scenario starts interactive shell instance, which will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell command "docker info"
+     Then stdout of shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
+      And stdout of shell should not contain "Name: minishift"
+
+  Scenario: Starting minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift has state "Running"
+
+  Scenario: Setting Docker client using docker-env command
+     When executing in shell command "minishift docker-env" succeeds
```

Comment:
is 'command' not superfluous here and "When executing in shell  "minishift docker-env" succeeds" easier to read?

Reply:
@hferentschik +1 much cleaner, will implement it, thanks